src/bindings/python/src/pyopenvino/graph/op_extension.hpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+
+#include "openvino/core/op_extension.hpp"
+#include "pyopenvino/graph/op.hpp"
+
+namespace py = pybind11;
+
+class PyOpExtension : public ov::BaseOpExtension {
```

Comment:
I think that if it's possible, implementation details for `PyOpExtension` should be moved to `op_extension.cpp`

Reply:
done